components/org.wso2.carbon.database.utils/pom.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>carbon-utils</artifactId>
+        <groupId>org.wso2.carbon.utils</groupId>
+        <version>2.0.5-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.wso2.carbon.utils</groupId>
+    <artifactId>org.wso2.carbon.database.utils</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jacoco</groupId>
+            <artifactId>org.jacoco.agent</artifactId>
+            <classifier>runtime</classifier>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>findbugs</groupId>
+            <artifactId>findbugs</artifactId>
+            <version>1.0.0</version>
```

Comment:
refer versions through properties.

Reply:
Why adding findbugs manually? It is coming from carbon-parent

Reply:
There was a findbugs warning called SQL_PREPARED_STATEMENT_GENERATED_FROM_NONCONSTANT_STRING in JdbcTemplate. So I had to suppress that warning by adding  @edu.umd.cs.findbugs.annotations.SuppressWarnings to the method. To add those annotations I need that dependency.